### PR TITLE
test(browser): add unit test cases for LogtoClient getters

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Add unit test cases for LogtoClient information getter functions
* getAccessToken
* getIdTokenClaims
* fetchUserInfo

## Testing

- [x] Passed all test cases

### CODE REVIEW

File: packages/browser/src/index.test.ts [hunk 1]
```diff
@@ -144,4 +159,81 @@ describe('LogtoClient', () => {
...
     });
   });
+
+  describe('getAccessToken', () => {
+    test('should throw if idToken is empty', async () => {
+      localStorage.removeItem(idTokenStorageKey);
+      localStorage.setItem(refreshTokenStorageKey, 'refresh_token_value');
+      const logtoClient = new LogtoClient({ endpoint, clientId }, requester);
+
+      await expect(logtoClient.getAccessToken()).rejects.toThrow();
```
Comment (IceHe) [inline packages/browser/src/index.test.ts:169]: better use `toMatchError` to match specific Error `new LogtoClientError('not_authenticated')`

Comment (demonzoo) [reply packages/browser/src/index.test.ts:169]: ok

**Review by simeng-li — APPROVED** [review]
LGTM

**Review by IceHe — CHANGES_REQUESTED** [review]
There is a bug in `getAccessToken` ( found in the unit test in #181 ).
See the commit https://github.com/logto-io/js/pull/181/commits/59b0516e249c069935ee4781924a8b348ed92e65 in #181
Please have a try after picking this commit or rebasing this branch.

**Review by wangsijie — APPROVED** [review]

**Review by IceHe — APPROVED** [review]
LGTM

**Pull request merged**